servlet/spring-boot/java/oauth2/azure-active-directory-b2c/README.adoc

```diff
@@ -0,0 +1,74 @@
+= Azure Active Directory B2C with Spring Security Sample
+
+In the current folder, we have the these samples to introduce how to use the Azure Active Directory B2C functions through Spring Security.
+
+The following sections provide the samples for Azure Active Directory B2C work with Spring Security.
+
+* <<work-as-web-application, Work as Web Application>>
+* <<work-as-resource-server, Work as Resource Server>>
+
+[[create-azure-active-directory-B2C-resources]]
```

Comment:
Before create resource, we should tell customer how to get a Azure account. I mean we need a `prerequisites`

Example:

https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/spring/ENVIRONMENT_CHECKLIST.md#prerequisites